app/context-map/workload.js

```diff
@@ -0,0 +1,20 @@
+const Workload = require('../points/domain/workload')
+const mapToTiers = require('./tiers')
+const Locations = require('../../app/staging/constants/locations')
+
+module.exports = function (communityTierCounts, custodyTierCounts, licenseTierCounts) {
+  var custodyTiersObject = mapToTiers(Locations.CUSTODY,
```

Comment:
Would we not be better using the enum that we will use to store it in the database though?